packages/rsocket-examples/src/RequestResponseExampleTcpClient.ts

```diff
@@ -0,0 +1,44 @@
+import { RSocketConnector } from "@rsocket/rsocket-core";
+import { TcpClientTransport } from "@rsocket/rsocket-tcp-client";
+import { exit } from "process";
+
+async function main() {
+  const connector = new RSocketConnector({
+    transport: new TcpClientTransport({
+      connectionOptions: {
+        host: "127.0.0.1",
+        port: 9090,
+      },
```

Comment:
`connectionOptions` could just be on the top-level options object, rather than nested, however, I thought it might be nice to segment the connection level options from other options that may be related to debug logging, etc. Ultimately I think we'll just want the various transports to have a consistent API.